### ISSUE TITLE
Cannot attach lspoints-copilot

### ISSUE THREAD

**Issue by tani** [kuuote/lspoints#20]
# Configuration

```vim
"Steps:
"$ export NVIM_APPNAME=nvim_lspoints
"$ mkdir ~/.config/$NVIM_APPNAME
"$ git clone https://github.com/vim-denops/denops.vim/
"$ git clone https://github.com/kuuote/lspoints
"$ git clone https://github.com/4513ECHO/lspoints-copilot
"$ nvim
"
"Run :echo lspoints#get_clients()
let &rtp..=expandcmd(',<sfile>:p:h/denops.vim')
let &rtp..=expandcmd(',<sfile>:p:h/lspoints')
let &rtp..=expandcmd(',<sfile>:p:h/lspoints-copilot')
call lspoints#load_extensions(['copilot'])
```

# Expected behavior

Neovim shows `['copilot']`.

# Actual behavior

Neovim showed `[]`.

ping @4513ECHO @kuuote . Thanks

**Comment by Shougo** [comment]
Attach is not included in your instruction.

**Comment by Shougo** [comment]
```vim
set rtp+=~/work/denops.vim
set rtp+=~/src/lspoints
set rtp+=~/src/lspoints-copilot
```

It just works.  `:Copilot` command is defined.  What is the problem?

**Comment by Shougo** [comment]
Hm... `load_extension()` is needed.
(The command is defined though)

If I remove it:

![2024-10-25_17-53](https://github.com/user-attachments/assets/7ed11bbf-3424-4f29-b54c-c9ed46b508d5)

**Comment by Shougo** [comment]
I cannot reproduce your behavior.  `copilot` extension is loaded and attached.

**Comment by 4513ECHO** [comment]
What the version of Deno is old causes this issue.

**Comment by tani** [comment]
This issue is solved by Deno 2.0.0. Thanks!
